src/NUnitTestExtractor/NUnitTestExtractor/NUnitTestExtractorApp.cs

```diff
@@ -148,10 +152,20 @@ public static void GetTests(IList<string> dlls, Level level, TextWriter writer)
                                         case Level.Function:
                                             data = methodInfo.DeclaringType + "." + methodInfo.Name;
                                             break;
+
+                                        case Level.TestCase:
+                                            
+                                            if(testCase != null && testCase.Arguments.Length > 0)
```

Comment:
This doesn't include TestCases with no parameters.  If you add this else if, it works:
```
else if (testCase != null && testCase.Arguments.Length == 0)
{
    data = methodInfo.DeclaringType + "." + methodInfo.Name + "()";
}
```

Then you also need a unit test for that scenario also.